overview/dashboard.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Dashboard"
+---
+
+PropelAuth provides signup, login, profile/account, and team management pages
+so you don’t have to worry about building them. These will also handle advanced cases like enrolling users in
+2FA, collecting additional metadata, and more.
+
+PropelAuth's prebuilt dashboard is a management tool that collects everything you need to configure your authentication preferences, users, and organizations, all in one place.
```

Comment:
instead of collects, maybe just has? "has everything you need"

And then maybe "you and your team need"

What do you think about "manage" before users and organizations as well?

Reply:
"PropelAuth's prebuilt dashboard is a management tool that has everything you and your team need to configure your authentication preferences, manage users, and configure organizations, all in one place." Is what I went with? What do you think?

Reply:
Yup that seems great!